### DISCUSSION TITLE
32 bit plugin for 32 bit Windows Cubase?

### DISCUSSION THREAD

**Discussion by rawl747** [apohl79/audiogridder#681] [Ideas]
I have a good friend that still uses a 32 bit version of Cubase.  JBridge used to work on his Windows 10 machine to allow him to use newer 64 bit plugins in his 32 bot Cubase but now because of Windows 10 updates, his 64 bit bridged plugins only show a blank white window when instantiated in Cubase.

A compile to a 32 bit Windows VST plugin that could "talk" to the existing 64 bit AudioGridder Server would be a life saver to my friend.  How hard would it be to complie the plugin to a 32 bit VST DLL file?

**Comment by apohl79** [comment]
Just compiling it for 32bit does not work. It likely requires a lot of code changes. But I have not looked at the details.

**Reply by rawl747** [reply]
OK, I had hoped it might be easier than that.  

I personally use Cubase Pro v11 and VEP v7 when necessary to bridge some older 32 bit plugins into that 64 bit environment. So, I expect that a 64 bit VEP server (slave) used with the 32 bit VEP plugin would work for him but VEP is rather pricey for a musician that has trouble finding the money to update his Cubase and his host of older 32 bit plugins.

Still, my friend is not alone.  Many amateur musicians are still stuck in a largely 32 bit reality while the world of plugins keeps moving on with 64 bit only plugins.

If you decide to recode and compile 32 bit versions, you may be surprised to find some takers.

Thank you for responding so quickly!

**Reply by e-d-i-t** [reply]
Well, you just answered my search... How to get 32bit VSTi's which are a lot (even new opensource stuff) working in some form other than jBridge with Cubase 13/14...
Just downloaded AudioGridder to find out, but it seems I don't need to install it anymore. (well, ofcourse I will, I still have some plugins which can use some extra cpu from another system...)
